### PR TITLE
:recycle: refactor select token icon

### PR DESCRIPTION
**Pull request by MarioBatalha**
---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1206583341563277

### CODE REVIEW

**Comment by git-marcopitra** [comment]
image
It's supposed to have the same size

**Comment by git-marcopitra** [comment]
And there is a bug when I select **FUD** the pug

**Review by KipandaJr — CHANGES_REQUESTED** [review]
Token selected  image
Token on Input 
image


Its is expected?

**Comment by MarioBatalha** [comment]
> Token selected image Token on Input image
> 
> Its is expected?

I didn't format the value, it's comes this way from the api.

File: constants/index.ts [hunk 1]
```diff
@@ -1,3 +1,4 @@
 export * from './dapp';
 export * from './routes';
 export * from './wrapper-variants';
+export type CHAIN = 'ETH' | 'BSC' | 'SOL' | 'AVA';
```
Comment (git-marcopitra) [inline constants/index.ts:4]: move to `@/interface` and transform type to PascalCase

**Review by git-marcopitra — CHANGES_REQUESTED** [review]

**Review by KipandaJr — APPROVED** [review]

**Review by git-marcopitra — APPROVED** [review]

**Review by atilasousa — APPROVED** [review]

**Review by git-marcopitra — APPROVED** [review]

**Pull request merged**